rust/Makefile.am

```diff
@@ -42,12 +42,30 @@ else
 			--features "$(RUST_FEATURES)"
 endif
 
+if HAVE_RUST_HEADERS
+
+all-local: build-rust
+	$(info Rust headers available)
```

Comment:
Will this only work on GNU make?

Reply:
> Will this only work on GNU make?

While it doesn't fail on FreeBSD, it doesn't print anything either. Probably best to stick with @echo...